packages/gatsby-plugin-cms/src/gatsby-node.ts

```diff
@@ -100,55 +105,74 @@ export const sourceNodes = async (
   gatsbyApi: SourceNodesArgs,
   options: Options
 ) => {
-  // Warning: Do not source remote and local nodes in a different order since this
-  // is important for the local nodes not to overrider remote ones
-  const nodes = await Promise.all([
-    fetchAllRemoteNodes(gatsbyApi, options),
-    fetchAllLocalNodes(gatsbyApi),
-  ]).then(([x, y]) => [...x, ...y].filter(Boolean))
+  const { webhookBody }: any = gatsbyApi
 
-  createCmsSchemaCustomization(gatsbyApi, nodes)
+  // Preview was triggered
+  if (webhookBody?.id && webhookBody.contentType) {
```

Comment:
webhookBody will only exist in previews?

Reply:
yes